packages/cli/src/util/build/monorepo.ts

```diff
@@ -50,9 +50,7 @@ export async function setMonorepoDefaultSettings(
     const { monorepoManager, ...commands } = result;
 
     output.log(
-      `Automatically detected ${title(
-        monorepoManager
-      )} monorepo manager. Attempting to assign default settings.`
```

Comment:
Curious though, is "Attempting" meaning it doesn't always set the defaults?

Reply:
yes

Reply:
See the `setCommand` method in the same file